### PR TITLE
Add FH 215

### PR DESCRIPTION
**Pull request by simonjulianl**
1. Add `nofh:true` by default to remove hangar footer 
2. Fix Friday Hacks schedule

### CODE REVIEW

File: data/friday_hacks.yml [hunk 1]
```diff
@@ -36,8 +28,8 @@ hacks:
         title: TBA
       - speaker: Sheng You
         title: Ktor in Action
-  - nohack: true
-  - nohack: true
+  - nohack: Recess Week
+  - nohack: Exam Week
   - venue: TBD (Physical)
```
Comment (kouyk) [inline data/friday_hacks.yml:33]: really minor nitpick but physical probably doesn't need to be indicated, since it is TBD anyway

**Review by kouyk — APPROVED** [review]
I'm just wonder if the zoom links should be replaced with youtube links after the event, but that's not really relevant here anyway

File: data/friday_hacks.yml (hunk 1 above)
Comment (simonjulianl) [reply data/friday_hacks.yml:33]: Yeah sure!

**Comment by simonjulianl** [comment]
I can change the zoom link to youtube link for future events, for past hnr its never changed but maybe its a good time to change it next time

**Pull request merged**